analyzers/tests/SonarAnalyzer.UnitTest/Rules/OptionStrictOnTest.cs

```diff
@@ -0,0 +1,34 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2021 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarAnalyzer.Rules.VisualBasic;
+using SonarAnalyzer.UnitTest.TestFramework;
+
+namespace SonarAnalyzer.UnitTest.Rules
+{
+    [TestClass]
+    public class OptionStrictOnTest
+    {
+        [TestMethod]
+        [TestCategory("Rule")]
+        public void OptionStrictOn() => Verifier.VerifyAnalyzer(@"TestCases\OptionStrictOn.vb", new OptionStrictOn());
```

Comment:
Let's add 2nd test with `Option Strict Off`
Let's add 3rd test with `Option Explicit Off` to be sure that we don't trigger on that one.

Reply:
I used these test names for the `Option Explicit` variant S6146:
```
OptionExplicitOn_IsOff
OptionExplicitOn_IsOn
OptionExplicitOn_IsOmissing
```